frontend/src/pages/AccountSend.vue

```diff
@@ -400,6 +486,7 @@ function useSendForm() {
       resetForm();
     } finally {
       isSending.value = false;
+      showAdvancedSendWarning = false;
```

Comment:
Also you define `let showAdvancedSendWarning = ref(false);` but it should be `const showAdvancedSendWarning = ref(false);`
```suggestion
      showAdvancedSendWarning.value = false;
```